cli_reference/basic_cli_operations.adoc

```diff
@@ -202,10 +202,29 @@ $ oc project <project_name>
 Return a list of objects for the specified xref:object-types[object type]. If
 the optional `<object_name>` is included in the request, then the list of
 results is filtered by that value.
+
 ----
 $ oc get <object_type> [<object_name>]
 ----
 
+The `oc get <object_type>` provides the `-o`, `--output` options to modify the output format.
```

Comment:
I'm not sure if `oc get` is _providing_ `-o`, and we're in the `get` section, so It might be better to say:

"You can use the `-o` or `--output` option to modify the output format."